vortexasdk/config.py

```diff
@@ -1,4 +1,9 @@
 import os
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
 
 LOG_LEVEL = os.getenv("LOG_LEVEL", "INFO")
 LOG_FILE = os.getenv("LOG_FILE", "vortexasdk.log")
+
+END_OF_AVAILABLE_DATA = datetime.utcnow() + relativedelta(months=6)
+BEGINNING_OF_AVAILABLE_DATA = END_OF_AVAILABLE_DATA - relativedelta(years=4)
```

Comment:
Are we time-boxing the data available to be loaded, even if clients use `load_all`?

Reply:
Yep, the API limits requests to 4 years max.

Ahh we could actually loop through batches of the min and max allowed date from the API